pkg/backend/display/tree.go

```diff
@@ -189,7 +189,7 @@ func (r *treeRenderer) markDirty() {
 	}
 
 	r.dirty = true
-	if r.opts.DeterministicOutput {
+	if r.opts.DeterministicOutput && r.opts.RenderOnDirty {
```

Comment:
is this extra flag meant to suppress output until the final frame?

Reply:
Yeah